android/app/build.gradle

```diff
@@ -22,7 +22,7 @@ if (flutterVersionName == null) {
 }
 
 apply plugin: 'com.android.application'
-apply plugin: 'com.google.gms.google-services'
+// apply plugin: 'com.google.gms.google-services'
```

Comment:
Don't do changes unrelated to PR's description. If you have build problems, file another issue